### PR TITLE
Get Constitution Sign it, no persistance

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

File: packages/ethapi/src/ethapi.ts [hunk 1]
```diff
@@ -136,8 +136,12 @@ export class EthApi {
     let signature: Hex;
     try {
       signature = (await this.rpc("personal_sign", messageHex, signerAccount, "")).result;
-    } catch {
-      signature = (await this.rpc("eth_sign", signerAccount, messageHex)).result;
+    } catch (e) {
+      if (e.message === "Error: MetaMask Message Signature: User denied message signature.") {
+        throw e; // rethrow the metamask error to be handled in ui
```
Comment (tobek) [inline packages/ethapi/src/ethapi.ts]: :ok_hand:

Comment (tobek) [inline packages/ethapi/src/ethapi.ts]: :ok_hand:

Comment (tobek) [reply packages/ethapi/src/ethapi.ts]: Though shame/weird there's no code or anything that might be more stable than this message.

Comment (walfly) [reply packages/ethapi/src/ethapi.ts]: yeah

**Pull request merged**